sale_timesheet_invoice_description/tests/test_sale_timesheet_description.py

```diff
@@ -8,12 +8,12 @@
 from odoo.tools.float_utils import float_compare
 
 
-class TestSaleTimesheetDescription(common.SavepointCase):
+class TestSaleTimesheetDescription(common.TransactionCase):
     @classmethod
     def setUpClass(cls):
         super(TestSaleTimesheetDescription, cls).setUpClass()
         # Make sure user is in English
-        cls.env["res.lang"].load_lang("en_US")
+        cls.env["res.lang"]._activate_lang("en_US")
```

Comment:
There's no need of loading this language, as it's already pre-loaded.

Reply:
Fixed